docs/advanced/input_files/input-main.md

```diff
@@ -422,7 +422,7 @@
     - [lr\_nstates](#lr_nstates)
     - [abs\_wavelen\_range](#abs_wavelen_range)
     - [out\_wfc\_lr](#out_wfc_lr)
-[back to top](#full-list-of-input-keywords)
```

Comment:
why the line 425 has been deleted?

Reply:
@mohanchen not sure... this is done by precommit bot, I will check if I can recover it, thanks for pointing out this!

Reply:
> @mohanchen not sure... this is done by precommit bot, I will check if I can recover it, thanks for pointing out this!

@haozhihan Hi, can you configure the bot so that it will ignore the markdown file? Here is an unexpected change brought about by it.

Reply:
I know why this happen... this is because there is a keyword absent in this list:
![image](https://github.com/user-attachments/assets/970ee8f8-d43f-4ad4-9095-28123fbbdb6e)
the keyword `abs_broadening` indeed exists but was forgotten to add to the list.